frontend/src/pages/landing_page/LandingPage.js

```diff
@@ -0,0 +1,13 @@
+import NavBar from "../../components/NavBar/NavBar";
+import WholeBody from "../../components/Body/WholeBody/WholeBody";
+import Footer from "../../components/Footer/Footer";
+
+export default function LandingPage() {
+    return (
+        <>
+            <NavBar />
+            <WholeBody />
```

Comment:
let's rename this component to something more useful but otherwise, this component is 👌